internal/runners/projects/projects.go

```diff
@@ -52,31 +73,36 @@ func (o *projectsOutput) MarshalOutput(f output.Format) interface{} {
 		Organization   string
 		LocalCheckouts string `locale:"local_checkouts,Local Checkouts" opts:"emptyNil,separateLine"`
 		Executables    string `locale:"executables,Executables" opts:"emptyNil,separateLine"`
+		LastUsed       string `locale:"last_used,Last Used" opts:"emptyNil,separateLine"`
 	}
 
 	r := []projectOutputPlain{}
 	for _, v := range o.projects {
 		checkouts := []string{}
 		executables := []string{}
+		lastUseds := []string{}
 		for i, checkout := range v.LocalCheckouts {
 			if len(v.Executables) <= i {
 				continue // remote project listings do not have executables
 			}
 			execDir := v.Executables[i]
-			if execDir != "" {
+			lastUsed := v.LastUsed[i]
+			if execDir != "" || lastUsed != "" {
```

Comment:
It seems like execDir being empty and lastUsed being non-empty is an impossible scenario. Is this supposed to be `&&`? If so we're still missing the `execDir!='' && lastUsed==''` scenario.

Reply:
You're right that the conditional is now superfluous.